core/src/test/java/de/unistuttgart/iaas/amyassist/amy/core/registry/ContactRegistryTest.java

```diff
@@ -0,0 +1,38 @@
+package de.unistuttgart.iaas.amyassist.amy.core.registry;
+
+import de.unistuttgart.iaas.amyassist.amy.core.plugin.api.registry.Contact;
+import de.unistuttgart.iaas.amyassist.amy.core.registry.contact.ContactRegistry;
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.MatcherAssert.*;
+import static org.hamcrest.Matchers.*;
+
+public class ContactRegistryTest {
+
+    ContactRegistry cr = new ContactRegistry();
```

Comment:
You MUST use the Test Framework to test Services

Reply:
I know - I just didn't get it working with the test framework yet. I don't see any way to test the whole registry system (including hibernate and H2 integration) without modifying our test framework.
I also can't just test this unit for now because I'm not sure whether I'm using the EntityManager correctly (yet).